tests/integration/garfish/main/src/App.tsx

```diff
@@ -58,9 +58,9 @@ const App: React.FC = () => {
             }}
           />
         </Route>
-        <Route path="/shared" exact={false}>
+        {/* <Route path="/shared" exact={false}>
```

Comment:
Why comment this

Reply:
DashboardButton need to be loaded by  ModuleFederationPlugin, but now, ModuleFederationPlugin not supported in Rspack.